locuszoom.app.js

```diff
@@ -437,13 +452,17 @@ LocusZoom.DataSources.prototype.getSource = function(ns) {
     return this.sources[ns];
 };
 
+LocusZoom.DataSources.prototype.removeSource = function(ns) {
+    return this.setSource(ns, null);
+};
+
 LocusZoom.DataSources.prototype.setSources = function(x) {
```

Comment:
The use case is basically serialization (which we aren't really using yet in our simple demo page). It allows you to set all the sources at once with a JSON object (or string). It's the inverse of the `JSON.stringify()` for the `DataSources` collection.